extensions/sql-database-projects/src/typings/vscode-mssql.d.ts

```diff
@@ -288,86 +288,113 @@ declare module 'vscode-mssql' {
-        ignoreIndexPadding: boolean;
-        ignoreIndexOptions: boolean;
-        ignoreIncrement: boolean;
-        ignoreIdentitySeed: boolean;
-        ignoreUserSettingsObjects: boolean;
-        ignoreFullTextCatalogFilePath: boolean;
-        ignoreWhitespace: boolean;
-        ignoreWithNocheckOnForeignKeys: boolean;
-        verifyCollationCompatibility: boolean;
-        unmodifiableObjectWarnings: boolean;
-        treatVerificationErrorsAsWarnings: boolean;
-        scriptRefreshModule: boolean;
-        scriptNewConstraintValidation: boolean;
-        scriptFileSize: boolean;
-        scriptDeployStateChecks: boolean;
-        scriptDatabaseOptions: boolean;
-        scriptDatabaseCompatibility: boolean;
-        scriptDatabaseCollation: boolean;
-        runDeploymentPlanExecutors: boolean;
-        registerDataTierApplication: boolean;
-        populateFilesOnFileGroups: boolean;
-        noAlterStatementsToChangeClrTypes: boolean;
-        includeTransactionalScripts: boolean;
-        includeCompositeObjects: boolean;
-        allowUnsafeRowLevelSecurityDataMovement: boolean;
-        ignoreWithNocheckOnCheckConstraints: boolean;
-        ignoreFillFactor: boolean;
-        ignoreFileSize: boolean;
-        ignoreFilegroupPlacement: boolean;
-        doNotAlterReplicatedObjects: boolean;
-        doNotAlterChangeDataCaptureObjects: boolean;
-        disableAndReenableDdlTriggers: boolean;
-        deployDatabaseInSingleUserMode: boolean;
-        createNewDatabase: boolean;
-        compareUsingTargetCollation: boolean;
-        commentOutSetVarDeclarations: boolean;
-        blockWhenDriftDetected: boolean;
-        blockOnPossibleDataLoss: boolean;
-        backupDatabaseBeforeChanges: boolean;
-        allowIncompatiblePlatform: boolean;
-        allowDropBlockingAssemblies: boolean;
-        dropConstraintsNotInSource: boolean;
-        dropDmlTriggersNotInSource: boolean;
-        dropExtendedPropertiesNotInSource: boolean;
-        dropIndexesNotInSource: boolean;
-        ignoreFileAndLogFilePath: boolean;
-        ignoreExtendedProperties: boolean;
-        ignoreDmlTriggerState: boolean;
-        ignoreDmlTriggerOrder: boolean;
-        ignoreDefaultSchema: boolean;
-        ignoreDdlTriggerState: boolean;
-        ignoreDdlTriggerOrder: boolean;
-        ignoreCryptographicProviderFilePath: boolean;
-        verifyDeployment: boolean;
-        ignoreComments: boolean;
-        ignoreColumnCollation: boolean;
-        ignoreAuthorizer: boolean;
-        ignoreAnsiNulls: boolean;
-        generateSmartDefaults: boolean;
-        dropStatisticsNotInSource: boolean;
-        dropRoleMembersNotInSource: boolean;
-        dropPermissionsNotInSource: boolean;
-        dropObjectsNotInSource: boolean;
-        ignoreColumnOrder: boolean;
-        doNotDropObjectTypes: SchemaObjectType[];
-        excludeObjectTypes: SchemaObjectType[];
-    }
+	export interface DacDeployOptionPropertyBoolean {
```

Comment:
also update this file in [vscode-mssql](https://github.com/microsoft/vscode-mssql) and update the STS version after the change gets checked into STS